15_in_words/logic.rb

```diff
@@ -0,0 +1,35 @@
+class Logic
```

Comment:
Why create a new class? Are you going to re-use these constants somewhere else?

Also `Logic` is a very generic term, you could use it almost everywhere. Try finding more specific names. :)